tests/notebooks/utils/test_evaluation.py

```diff
@@ -0,0 +1,117 @@
+# tests/notebooks/utils/test_evaluation.py
+
+import json
+import pytest
+from notebooks.utils.evaluation import PhenotypeEvaluator
+from notebooks.utils.phenopacket import Phenopacket
+
+
+@pytest.fixture
+def path_to_true_phenopacket(tmp_path):
+    """
+    Create a minimal ground-truth Phenopacket JSON file with two HPO labels:
+      - "Phen1"
+      - "Phen2"
+    Return the path to that file.
+    """
+    data = {
+        "phenotypicFeatures": [
+            {"type": {"id": "HP:0000001", "label": "Phen1"}},
+            {"type": {"id": "HP:0000002", "label": "Phen2"}},
+        ]
+    }
+    p = tmp_path / "true_packet.json"
+    p.write_text(json.dumps(data), encoding="utf-8")
+    return str(p)
```

Comment:
Is there a reason, why you need to dump the json?
cant you just return the string itself?